src/coldfront_plugin_api/tests/unit/test_allocations.py

```diff
@@ -18,6 +19,17 @@ def setUp(self) -> None:
 
         self.resource = self.new_resource(name="Devstack", auth_url="http://localhost")
 
+    @staticmethod
+    def new_allocation_attribute(allocation, attribute, value):
```

Comment:
Could [this](https://github.com/nerc-project/coldfront-plugin-cloud/blob/114fd83de5b818f05f10317b4f299d9083e2248e/src/coldfront_plugin_cloud/utils.py#L21) method have worked? Why did you write a new one?

Reply:
I realized that the utility function in `coldfront_plugin_cloud` didn't return the allocation attribute, which I need to perform assertions. I could probably still use that function, but it might result in messier assertions.